src/renderer/utils/format.tsx

```diff
@@ -1,14 +1,19 @@
 import dayjs from 'dayjs';
 import relativeTime from 'dayjs/plugin/relativeTime';
+import utc from 'dayjs/plugin/utc';
 import formatDuration from 'format-duration';
 import type { Album, AlbumArtist, Song } from '/@/renderer/api/types';
 import { Rating } from '/@/renderer/components/rating';
 
 dayjs.extend(relativeTime);
+dayjs.extend(utc);
 
 export const formatDateAbsolute = (key: string | null) =>
```

Comment:
Might as well make the string a const and use it in both rather than duplicate

Reply:
Alright, cleaned that up.